### ISSUE TITLE
[Accessibility] Measure properties not in context menu for keyboard users

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#19509]
### Issue type

UI bug

### Bug description

There is no way to select an empty area of music with the keyboard and therefore no way to get to measure properties using the menu key.

### Steps to reproduce

Right click on any empty part of a score to see the list of available options including measure properties.  With NVDA enabled arrow on to a note or any other item and press the menu key (next to right Ctrl) and only options for those notes appear.  Selecting an area does not seem to work either with this menu key.

### MuseScore Version

4

### Regression

I don't know

### Operating system

Windows 10

### Additional context

Related to:
https://github.com/musescore/MuseScore/issues/15379

**Comment by shoogle** [comment]
> There is no way to select an empty area of music with the keyboard

Indeed. However, it *is* possible to create a *range selection* via the keyboard with Shift + Right arrow. For sighted users, the context menu that appears when you right-click on a range selection is the same as the one that appears when you right click on an empty area of the staff. It contains these options after cut, copy, paste & swap with clipboard:

- Clear measures
- Delete measures
- Staff/Part properties
- Insert measures
- Measure properties

However, the menu contains different options when it is triggered via the keyboard. So I think the solution here is to make the keyboard context menu for range selections equivalent to the mouse context menu for range selections.

Right-click on range selection | Menu key or Shift+F10 on range selection
---|---
![](https://github.com/musescore/MuseScore/assets/11011881/a6f6ad85-6f25-4e08-8809-14f959dfdccd) | ![](https://github.com/musescore/MuseScore/assets/11011881/0d7668dd-4d71-4dee-90f3-19cf2ae2c625)

<details><summary><b>Alternative solutions <i>(click to show/hide)</i></b></summary>

1. We could expose those menu options in the context menu for notes and rests, so keyboard users can select any note, press the menu key, and then access those options. However, there are other options needed in the note and rest context menu, so adding all these additional ones would make the menu very long indeed.

3. We could expose `Measure` in the Alt+Right navigation tree, so that the first element in each measure is the measure itself. However, if you're in the middle of a long measure you could be navigating for a while before you reach the next `Measure` object and are able to trigger the menu. Also, if you're not trying to trigger the menu, all those extra `Measure` objects would be useless and only serve to get in the way of normal navigation.

</details>

**Comment by jmwmusic** [comment]
I think being able to make a selection and use the menu key to open the same menu items as a right click would be an excellent solution.  Of the suggestions above this would seem the most natural to a screen reader.  Often when looking up how to do something we can only find the sighted version and then have to interpret the most similar way we could do this.

**Comment by DmitryArefiev** [comment]
and #19279 still serious for blind users

**Comment by NinjaNas** [comment]
@DmitryArefiev https://github.com/musescore/MuseScore/issues/15379 should be closed too, I believe.
